processingchain/src/main/groovy/org/orbisgis/orbisprocess/geoclimate/processingchain/Workflow.groovy

```diff
@@ -1568,7 +1568,7 @@ def OSM() {
                 if(geoclimatedb){
                     def h2gis_path = geoclimatedb.get("path")
                     def delete_h2gis_db = geoclimatedb.get("delete")
-                    if(delete_h2gis_db && delete_h2gis_db in Boolean){
+                    if((delete_h2gis_db && delete_h2gis_db) in Boolean){
```

Comment:
You think about 

if(delete_h2gis_db && (delete_h2gis_db in Boolean)){

Reply:
No this one give false.
Try the following example
```groovy
def test = false
def geoclimatedb = [
        "path" : "/tmp/vannes/geoclimate_db;AUTO_SERVER=TRUE",
        "delete" :false
    ]
delete_h2gis_db = geoclimatedb.get("delete")
if(delete_h2gis_db && delete_h2gis_db in Boolean){
	logger.info geoclimatedb.get("delete")
	}

logger.info (delete_h2gis_db && (delete_h2gis_db in Boolean))```